web/src/main/webapp/package.json

```diff
@@ -14,8 +14,8 @@
     "url": "git://github.com/mbudiu-vmw/hiero.git"
   },
   "dependencies": {
-    "@types/d3": "^4.9.0",
-    "d3": "^4.0.5",
+    "@types/d3": "^4.10.0",
```

Comment:
I'm not sure about what webpack will upgrade automatically, but this was the current version in npm when I configured the GUI.

I'm not using any new functionality, so we might as well use older versions. But I don't see any problems with using the newer version either.